### PR TITLE
Show query trace right in Grafana

### PR DESCRIPTION
**Pull request by Loori-R**
- #36

image

### CODE REVIEW

File: src/querybuilder/components/Trace/NestedNav.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+          totalMsec={totalMsec}
+        />
+      ))}
+    </div>
+  );
+};
+
+export default NestedNav;
+
+const getStyles = (theme: GrafanaTheme2) => ({
```
Comment (dmitryk-dk) [inline src/querybuilder/components/Trace/NestedNav.tsx:58]: nit: i think we can move styles near the component

**Review by dmitryk-dk — APPROVED** [review]
LGTM! Add not critical comment

File: src/querybuilder/components/Trace/NestedNav.tsx (hunk 1 above)
Comment (Loori-R) [reply src/querybuilder/components/Trace/NestedNav.tsx]: updated

**Review by hagen1778 — APPROVED** [review]
Great!

**Comment by hagen1778** [comment]
@Loori-R thank you for the PR!

What do you think, can we add a button to copy JSON version of the trace into user's buffer or show a popup window with JSON representation like we do in VMUI? The purpose is to let users option to share the trace for the query.

**Comment by Loori-R** [comment]
@hagen1778 Sure, I'll add it.

**Pull request merged**